mslib/mscolab/file_manager.py

```diff
@@ -39,7 +40,7 @@ class FileManager(object):
     def __init__(self, data_dir):
         self.data_dir = data_dir
 
-    def create_operation(self, path, description, user, content=None, category="default"):
+    def create_operation(self, path, description, user, last_used=datetime.datetime.utcnow(), content=None, category="default"):
         """
```

Comment:
let's change this into 
last_used=None

then later 
if last_used is None: 
    last_used = datetime.datetime.utcnow()